openlr/candidate_paths_getter.cpp

```diff
@@ -0,0 +1,348 @@
+#include "openlr/candidate_paths_getter.hpp"
+
+#include "openlr/candidate_points_getter.hpp"
+#include "openlr/graph.hpp"
+#include "openlr/helpers.hpp"
+#include "openlr/openlr_model.hpp"
+
+#include "routing/road_graph.hpp"
+
+#include "platform/location.hpp"
+
+#include "geometry/angles.hpp"
+
+#include <tuple>
+
+using namespace std;
+using namespace routing;
+
+namespace openlr
+{
+namespace
+{
+// TODO(mgsergio): Maybe add a penalty if this value deviates, not just throw it away.
+int const kFRCThreshold = 3;
```

Comment:
Either openlr/router.hpp or openlr/candidate_paths_getter.cpp will not be used in the future.
And because of that I don't thing this does not worth spending my time.

Reply:
And, by the way, may kFRCThreshold is different.